core/src/test/java/io/questdb/cairo/CreateTableTest.java

```diff
@@ -58,8 +58,7 @@ public void testCreateTableAsSelectIndexUnsupportedColumnType() throws Exception
                 "CREATE TABLE tab AS (" +
                         "SELECT x FROM long_sequence(1)" +
                         "), INDEX(x)",
-                0,
-                "indexes are supported only for SYMBOL columns: x"
+                0
```

Comment:
Does not look like a good change, why error text is excluded from the assertion?

Reply:
because it is inlined, the value of the error text argument was the same across multiple tests